Library/Formula/nesc.rb

```diff
@@ -11,7 +14,24 @@ class Nesc < Formula
     sha256 "e39de1a28a7459f5948826bb8533c4adf1e2efe76eb7230a1dd8de00eb625cb7" => :mavericks
   end
 
+  # The compilation procedure for nesc is a bit weird. Hopefully, this will not be necessary
+  # in the next version.
+  #
+  # The regular way to install is: bootstrap, configure, make, install. Unfortunately, emacs
+  # blocks during the install step in interactive mode and the installation just hangs.
```

Comment:
Emacs can be called interactively, this sounds like a bug that should be reported upstream.